subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -57,6 +57,14 @@ The dependencies for annotation processors are not added to an IDEA module's cla
 - PMD has been updated to https://pmd.github.io/pmd-6.21.0/pmd_release_notes.html#24-january-2020---6210[PMD 6.21.0].
 - CodeNarc has been updated to https://github.com/CodeNarc/CodeNarc/blob/v1.5/CHANGELOG.md#version-15----nov-2019[CodeNarc 1.5].
 
+==== Degraded experience when using Gradle on 32-bit operating systems
+
+Gradle uses the https://github.com/gradle/native-platform[`native-platform`] library to call native platform APIs for things which are not easily possible from plain Java.
+Most notably, it uses `native-platform` for a better console experience.
+
+Gradle 6.3 uses `native-platform` 0.22, which drops support for 32-bit Unix systems and for old FreeBSD versions (older than FreeBSD 10).
+While Gradle will still continue building your projects on that systems, you may have a worse user experience especially when it comes to console output.
```

Comment:
Again, too much technical detail. The cause of the change doesn't make a difference for the user, as they can't do anything about it. Why not just write:

```suggestion
Gradle 6.3 does not support the rich console for 32-bit Unix systems and for old FreeBSD versions (older than FreeBSD 10).
Gradle will continue building your projects on the affected systems, but without the rich console.
32-bit versions of Windows are unaffected, and will keep working as before.
```